.github/workflows/save_daily_metrics.yml

```diff
@@ -50,19 +50,31 @@ jobs:
 
       - shell: bash -l {0}
         name: Save Github Metrics
+        id: github
         env:
           API_TOKEN_GITHUB: ${{ secrets.USAGE_STATS_ACCESS_TOKEN }}
         run: |
           python src/usage_metrics/scripts/save_github_metrics.py
 
       - shell: bash -l {0}
         name: Save Kaggle Metrics
+        id: kaggle
+        # Run even if prior metric saving fails
+        if: "!cancelled()"
         env:
           KAGGLE_KEY: ${{ secrets.KAGGLE_KEY }}
           KAGGLE_USERNAME: ${{ secrets.KAGGLE_USERNAME }}
         run: |
           python src/usage_metrics/scripts/save_kaggle_metrics.py
 
+      - shell: bash -l {0}
```

Comment:
Non blocking but we could make use of the github action matrix strategy to run multiple archives in parallel like we do in the [pudl-archiver repo](https://github.com/catalyst-cooperative/pudl-archiver/blob/13bbba5e2fa295cd45552006593073e4e5ce7023/.github/workflows/run-archiver.yml#L31-L35).

Reply:
I'll add this suggestion to #175.